src/libsync/owncloudpropagator.cpp

```diff
@@ -574,20 +577,29 @@ OwncloudPropagator::DiskSpaceResult OwncloudPropagator::diskSpaceCheck() const
 
 // ================================================================================
 
+void PropagateDirectory::updateJobPredicateValues()
+{
+    QMutableVectorIterator<PropagatorJob *> containerJobsIterator(_containerJobs);
```

Comment:
Also, please mind that, you will never have more than 100 folders in one folder to sync..